app/javascript/controllers/manualDetection.js

```diff
@@ -209,4 +212,311 @@ wpd.dataPointLabelEditor = (function() {
         cancel: cancel,
         keydown: keydown
     };
-})();
\ No newline at end of file
+})();
+
+wpd.dataPointValueOverrideEditor = (function() {
+    var ds, ax, ptIndexes, tool;
```

Comment:
Prefer using `let` or `const` instead of `var`. I used `var` in some places in the code in the past - but that was before ES6 was a thing.

Reply:
ref: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75

Reply:
Gotcha. I was trying to keep it consistent within the file, but happy to change all the `var`s to `let`s and `const`s.